### ISSUE TITLE
FEATURE: Add 'Unsaved Tab' to Global Styles

### ISSUE THREAD

**Issue by dcog989** [notepad-plus-plus/notepad-plus-plus#14254]
### FEATURE REQUEST

In order for users to style unsaved tabs and differentiate from saved tabs, please add to Style Configurator ... Global Styles:

* Active tab unsaved
* Inactive tab unsaved

**Comment by alankilborn** [comment]
The differentiation between saved and unsaved tabs has always been in the tab icon.  As the modern "alternate icon" set is very differentiating, I don't see any further change (such as the request here) happening.

**Comment by dcog989** [comment]
What is the "modern "alternate icon" set"?! You need to realise that not everyone knows everything that you do.

Providing this additional customisation option to users is in keeping with the vast array of customisation options already available in this program, and would be a valuable addition to many users. You don't need to use it if you don't want to!

**Comment by alankilborn** [comment]
If you are going to request changes to the software, it's on you to understand its current workings.  See out _Alternate icons_ in the settings and see its effect.

Developers have limited time; since a mechanism for easily differentiating modified and unmodified tabs already exists, I doubt extra effort will be expended to duplicate that functionality in a different way.

**Comment by dcog989** [comment]
Ah! You meant the 'Alternate icons' switch in Settings - which has no bearing on the feature request made here as it does not allow the user to customize the tab.

Again, try to differentiate between your narrow requirements and the much wider requirements of many, many users who would appreciate this additional customization. Thankfully Don Ho controls this project and not you!!

**Comment by alankilborn** [comment]
>Ah! You meant the 'Alternate icons' switch in Settings 

Yes, which is why I referred to "Alternate icons" in the "Settings".  Hmm.

>which has no bearing on the feature request made here as it does not allow the user to customize the tab.

Make whatever very-specific feature request you want -- those are free and easy.

I was merely trying to point out that visual tab differentiation already exists in a fine (IMO) fashion -- and you might not have been aware of it.  Choose to use it today...or not, as you like.

**Comment by dcog989** [comment]
> which is why I referred to "Alternate icons" in the "Settings". Hmm.

Why are you lying? Everyone can read your first comment! You're making stuff up to try and dig yourself out of a losing hole.

I'm not sure why you are incapable of grasping that a feature that you won't use could be very useful to a lot of other users. Please stop commenting on my ticket.

**Comment by alankilborn** [comment]
🙄
